packages/core/src/icon/icon.element.scss

```diff
@@ -1,30 +1,24 @@
 @import './../styles/tokens/generated/index';
 @import './../styles/mixins/mixins';
 
-@mixin rotateSVGIcon($deg) {
-  svg {
-    transform: rotate($deg);
-  }
-}
-
 @mixin flipSVGIcon($direction) {
   @if $direction == 'horizontal' {
-    svg {
+    :host {
       transform: scale(-1) rotateX(180deg);
     }
     /* stylelint-disable */
   } @else if $direction == 'vertical' {
     /* stylelint-enable */
-    svg {
+    :host {
```

Comment:
I think this is safe to change. Switching the rotation to be on the host made it easier for a host component to apply a transition against the icon rather than having to drill down into the inner template with a custom prop.